qp_qiime2/__init__.py

```diff
@@ -0,0 +1,33 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+from .qiime2 import rarefy
+
+__all__ = ['qiime2']
+
+
+# Initialize the plugin
+plugin = QiitaPlugin(
+    'qiime2', '4.2017', 'QIIME 2')
```

Comment:
why change the version string? will this be compatible with `2017.4.x`? also, note, `2017.5` will be coming out this week